components/toggl/actions/get-time-entries/get-time-entries.mjs

```diff
@@ -0,0 +1,21 @@
+import toggl from "../../toggl.app.mjs";
+
+export default {
+  name: "Get Time Entries",
+  version: "0.0.1",
+  key: "tally-get-time-entries",
+  description: "Get the last thousand time entries. [See docs here](https://github.com/toggl/toggl_api_docs/blob/master/chapters/time_entries.md#get-time-entries-started-in-a-specific-time-range)",
```

Comment:
Could we add a `max` prop that defaults to 1000 in case the user would like to return less than 1000 entries?